source/creating_maps_charts/02_creating_advanced_charts/analyze.rst

```diff
@@ -48,91 +47,114 @@ Here is what this layer looks like with these settings and two thresholds:
 Adding a series
 ---------------
 
-You can add a data representation on the same dataset by clicking on ``Add a series``.
-
-The `X-Axis` parameters will be the same, but you can customize the type of curve and tweak the appearance parameters on the new series.
+You can add a data representation on the same dataset by clicking on **Add a series**.
 
-Try changing the type from ``Line`` to ``Spline`` to see a nice curve on top of the initial column chart.
+The **X-Axis** parameters will be the same, but you can customize the type of curve and tweak the appearance parameters on the new series.
 
-You can reorder the series by clicking and dragging. The series order affects the chart legend, and the series at the top has priority on the `Y-Axis` parameter.
+Try changing the type from **Line** to **Spline** to see a nice curve on top of the initial column chart.
 
-In the next section we will add another dataset to our chart.
+You can reorder the series by clicking and dragging. The series order affects the chart legend, and the series at the top has priority on the **Y-Axis** parameter.
 
 
 Stacking layers
 ---------------
 
-Let's elaborate on another data visualization, this time with multiple layers.
+You create data visualizations with multiple layers. Such visualizations can help you find trends in data.
 
-**First Layer**
 
-We will start the same way as before, click on ``Add a dataset to this layer``, then search for "Oil Prices" to add the `Oil Prices <https://public-us.opendatasoft.com/explore/dataset/oil-prices>`_ dataset.
+Creating a chart with multiple layers
+-------------------------------------
 
-No need to filter it, just note that the data starts from the year 1987. We will use 1987 as the starting point for the other layers.
+For example, you can create a data visualization to find whether oil prices are correlated with gold prices.
 
-When you first import a dataset in Advanced Charts, the default visualization of this dataset is imported. In this case, you can see two lines: `WTI Spot Price` and `Brent Spot Price`.
+This example of chart relies on the following datasets:
 
-**Second Layer**
+ - `Oil Prices <https://documentation-resources.opendatasoft.com/explore/dataset/oil-prices/>`__ that contains monthly oil prices from 1988 to 2019, and
+ - `Gold Prices <https://documentation-resources.opendatasoft.com/explore/dataset/gold-prices/>`__ that contains monthly gold prices from 1988 to 2019.
 
-Let's add another dataset! Look for "Commodity Prices" to find `Commodity Prices since 1980 <https://public.opendatasoft.com/explore/dataset/commodity-prices-since-1980>`_ dataset from the Opendatasoft public collection.
+This chart compares Brent and WTI prices with gold prices between 1989 and 2019.
 
-This time, add the filter "`date>1987`" so that the X-axis will match with the first layer. Then import the ~20 000 records.
+To follow this example, add these datasets as sources.
```

Comment:
- Hey Sanka, you dead ?
- Nah man, I'm just confused because of redundant instructions between line 77 and line 86.

This is probably a subjective comment but I read these lines like :
- add datasets as sources
- then add oil prices dataset
- then click on add a dataset and select oil prices dataset

IMHO, the simpler the better

Reply:
Thanks Coach!

I'm going to remove the first two items you mentioned and only keep the "click on add a dataset and select oil prices dataset" part.